keanu-project/src/main/java/io/improbable/keanu/templating/SequenceItem.java

```diff
@@ -56,6 +56,14 @@ public SequenceItem() {
         return v;
     }
 
+    /**
+     * @return Returns a map of vertex labels to vertices, which covers all of the vertices that have been explicitly
+     * added to this sequence item.
+     */
+    public Map<VertexLabel, Vertex<?>> getContents() {
+        return this.contents;
```

Comment:
Should this be an immutable copy?

Reply:
What would be the benefit of this?

Reply:
So a user can't mutate the contents with what's returned